opencv.go

```diff
@@ -193,12 +197,28 @@ func (f *Framebuffer) Fit(width, height int, dst *Framebuffer) error {
 	}
 
 	var left, top int
-	left = int(float64(f.width-widthPostCrop) * 0.5)
+	switch fitDir {
```

Comment:
Another option would be to allow the top/left to be specified. Then we can crop around any given x/y we want.